open-api/rest-catalog-open-api.py

```diff
@@ -1100,6 +1100,31 @@ class LoadTableResult(BaseModel):
      - `s3.session-token`: if present, this value should be used for as the session token
      - `s3.remote-signing-enabled`: if `true` remote signing should be performed as described in the `s3-signer-open-api.yaml` specification
```

Comment:
can we also add a key for expiration time for aws sts token ?

Reply:
@stirupati I would suggest splitting that out in a separate PR, so we can purely focus on GCS/ADLS for this PR.

Reply:
@Fokko created separate PR for the same https://github.com/apache/iceberg/pull/10873